source/frontend/src/components/maps/leaflet/Control/AdvancedFilter/FilterContentContainer.tsx

```diff
@@ -12,42 +10,27 @@ export interface IFilterContentContainerProps {
 }
 
 export const FilterContentContainer: React.FC<IFilterContentContainerProps> = ({ View }) => {
-  const { isFiltering, setVisiblePimsProperties, setShowDisposed, setShowRetired, resetMapFilter } =
-    useMapStateMachine();
-
-  const { getMatchingProperties } = usePimsPropertyRepository();
```

Comment:
this request logic needed to be moved out of here as this is a conditionally rendered component, and therefore will not run during map load. It needs to run during map load since the default filter takes effect immediately now.